tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/tuples/BasicTupleTest.java

```diff
@@ -243,6 +243,27 @@ public void testNegativesOfTupleType() {
         BAssertUtil.validateError(resultNegative, i, "list index out of range: index: '-1'", 165, 19);
     }
 
+    @Test(description = "Test tuple declared with var")
+    public void testTupleDeclaredWithVar() {
+        BRunUtil.invoke(result, "testTupleDeclaredWithVar1");
+        BRunUtil.invoke(result, "testTupleDeclaredWithVar2");
+        BRunUtil.invoke(result, "testTupleDeclaredWithVar3");
+        BRunUtil.invoke(result, "testTupleDeclaredWithVar4");
```

Comment:
Better to use a data provider so that the rest of the tests run even if one fails.